ci/trivy-markdown.tpl

```diff
@@ -0,0 +1,58 @@
+## Vulnerability Report by [Trivy](https://trivy.dev)
+
+<details>
```

Comment:
Given that GitHub automatically hides the content of each except the first few files (https://github.com/caponetto/opendatahub-io-notebooks/actions/runs/9713547448#summary-26810623673) it may make more sense to not hide this. Now I have to click "expand" twice, instead of once.

Reply:
Given that we don't control how many of them GitHub hides, I preferred to set them all collapsed so that you can expand only the ones you are interested in seeing, avoiding uncontrolled scrolling.